.flake8

```diff
@@ -9,4 +9,4 @@ arg-type-hints-in-signature = True
 arg-type-hints-in-docstring = False
 # Ignore documentation linting in tests
 per-file-ignores =
-    tests/**: DOC
+    tests/**: DOC, D
```

Comment:
What is `D` here?

Reply:
`D` is for `pydocstyle` rules. I think I accidentally removed ignoring it when was moving pydocstyle validation to ruff instead of flake8.